src/olympia/devhub/templates/devhub/includes/modal_rollback_version.html

```diff
@@ -1,27 +1,35 @@
 <div id="modal-rollback-version" class="modal modal-rollback">
     <h3>{{ _('Roll back to a previous version') }}</h3>
-    <div>
-        {% trans addon_type=addon.get_type_display() %}Users of this {{ addon_type }} will roll back to the previous version you choose.{% endtrans %}
-        <a href="">{{ _('Learn more') }}</a>
-    </div>
-
     <form method="post" action="#rollback-version">
         <table>
+        <tr>
+            <td colspan="3">
```

Comment:
Why `colspan="3"`? I see for one of the rows below you've added an empty cell, but then the other 2 rows further below still only have 2 cells.

Reply:
The `<td>&nbsp;</td>` on row 24 isn't necessary - html tables add spaces for additional columns that are missing in rows.  The `colspan="3"` is to have it looking like the screenshot - with an empty column on the right of the selectable fields, without the text rows being too squashed.

I was aiming for the mockup layout:
<img width="1133" height="1028" alt="Screenshot 2025-08-27 115734" src="https://github.com/user-attachments/assets/e2e40a0b-8cce-49d4-b250-94bedc4ce523" />

Suggestions of alternative ways of doing this welcome!